changelog/8.9.0.asciidoc

```diff
@@ -0,0 +1,86 @@
+version: 8.9.0
+entries:
+    - kind: security
+      summary: Use verified base image for building Fleet Server binaries
+      description: ""
+      component: ""
+      pr:
+        - "2339"
+      issue: []
+      timestamp: 1682696415
+      file:
+        name: 1682696415-crossbuild.yaml
+        checksum: 159350f6e8e2e60912b5ca57c2d0ee3fe10b1026
+    - kind: breaking-change
+      summary: API default error is now 500
+      description: Up until now when any not expected error was encountered it resulted int Bad Request response.\n This is changing and any not expected error will return Internal Server Error while keeping most of the current behavior\n unchanged and returning Bad Request on expected failure paths (e.g. Agent Inactive, Missing Agent ID, Missing Auth Header).
```

Comment:
replaced newlines with \n as the changelog tool complained, is there a way to check the rendered version?

Reply:
@juliaElastic  The output here looks to be yaml format rather than asciidoc. It may not matter though, since I just copy the contents into a separate docs PR. Here's what I have for this breaking change:

Asciidoc:
```
[discrete]
[[breaking-2531]]
.API default error code is now 500
[%collapsible]
====
*Details* +
Previously, when {fleet-server} encountered an unexpected error it resulted in a `Bad Request` response.

*Impact* +
Now, any unexpected error returns an `Internal Server Error` response while keeping most of the current behavior
unchanged. On expected failure paths (for example, Agent Inactive, Missing Agent ID, Missing Auth Header) a `Bad Request` response is returned. For more information, refer to {fleet-server-pull}2531[#2531].
====
```


Rendering:
![Screenshot 2023-07-20 at 11 33 26 AM](https://github.com/elastic/fleet-server/assets/41695641/1e8a3a4a-b465-43f7-abb8-523319d47bfc)